signal/proto/signalexchange.proto

```diff
@@ -49,4 +49,5 @@ message Body {
   }
   Type type = 1;
   string payload = 2;
+  uint32 port = 3;
```

Comment:
I think that we should be more explicit in calling this field.
E.g. WireGuard listen port or native port